metron-platform/metron-elasticsearch/src/main/java/org/apache/metron/elasticsearch/dao/ElasticsearchDao.java

```diff
@@ -375,79 +523,14 @@ protected UpdateRequest buildUpdateRequest(Document update, String sensorType, S
         .upsert(indexRequest);
   }
 
-  @SuppressWarnings("unchecked")
   @Override
   public Map<String, Map<String, FieldType>> getColumnMetadata(List<String> indices) throws IOException {
```

Comment:
I did not remove the `getColumnMetadata` from the `IndexDao` interface.  I simply refer to the implementation provided by the `ColumnMetadataDao` instance.